kmelia/kmelia-library/src/main/java/org/silverpeas/components/kmelia/workflowextensions/SendInKmelia.java

```diff
@@ -232,11 +232,10 @@ public void populateFields(String pubId, ForeignPK fromPK, ForeignPK toPK) {
           fieldValue = fieldOfFolder.getObjectValue();
           // Check file attachment in order to put them inside form
           if (fieldOfFolder instanceof FileField) {
-
             fieldValue = copyFormFile(fromPK, toPK, ((FileField) fieldOfFolder).getAttachmentId());
           } else if ("wysiwyg".equals(fieldTemplate.getDisplayerName())) {
             WysiwygFCKFieldDisplayer displayer = new WysiwygFCKFieldDisplayer();
-            fieldValue = displayer.duplicateContent(fieldOfFolder, fieldTemplate, fromPK, toPK,
+            fieldValue = displayer.duplicateContent(fieldTemplate, fromPK, toPK,
                 I18NHelper.defaultLanguage);
           }
         } catch (WorkflowException e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS00108)